### DISCUSSION TITLE
scrollIntoView speed

### DISCUSSION THREAD

**Discussion by mizok** [idiotWu/smooth-scrollbar#506] [Q&A]
Is there any way to customize `Scrollbar.scrollIntoView` speed (or duration)?
Sometime it just scrolls too fast 😭

**Comment by dolphin-wood** [comment]
In that case, you can use the [`.scrollTo()`](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbarscrollto) method with a custom easing function:

```js
const { left, top } = elem.getBoundingClientRect();

scrollbar.scrollTo(
  /* x */ left - scrollbar.bounding.left, 
  /* y */ top - scrollbar.bounding.top, 
  /* duration */ 300, 
  /* easing, callback */ {},
);
```
